imap_processing/tests/lo/test_lo_l1b.py

```diff
@@ -397,14 +398,18 @@ def test_set_coincidence_type(attr_mgr_l1a):
     l1a_de = xr.Dataset(
         {
             "de_count": ("epoch", [3]),
-            "coincidence_type": ("direct_event", [0, 0, 4]),
-            "mode": ("direct_event", [1, 0, 1]),
-            "tof0": ("direct_event", [5, 2, 10]),
-            "tof1": ("direct_event", [10, 4, tof_fill]),
-            "tof2": ("direct_event", [15, 6, 20]),
-            "tof3": ("direct_event", [20, 8, 30]),
-            "cksm": ("direct_event", [25, ckm_fill, ckm_fill]),
-        }
+            "coincidence_type": ("direct_events", [0, 0, 4]),
```

Comment:
I was wondering what changed here when first looking at this lol, took me a bit to figure out it was the plural.

Do you want to have this pluralized "events" versus "event" before? I think the previous version might be more standard to align with "epoch" being singular, and "time" being singular.

Reply:
I agree. I changed it here to match what I have in the yaml and L1A code, but I'll create an issue to change L1A to use make it singular.

#1613